platforms/extensibility/plugin-use/src/main/java/org/gradle/plugin/use/resolve/service/internal/DefaultInjectedClasspathPluginResolver.java

```diff
@@ -87,13 +106,6 @@ private PluginRegistry createPluginRegistry(Factory<DependencyResolutionServices
         );
     }
 
-    /**
-     * InstrumentationStrategy will report a problem if the agent is used with TestKit, see ConfigurationCacheInjectedClasspathInstrumentationStrategy class.
```

Comment:
💅 can we keep this comment somewhere around `getTransform` call?

Reply:
Done